amazon_payments/views.py

```diff
@@ -480,6 +481,7 @@ class AmazonOneStepPaymentDetailsView(BaseAmazonPaymentDetailsView):
         'check_basket_is_valid',)
 
     def get_default_shipping_method(self, basket):
```

Comment:
@sjolicoeur Are you sure that this function isn't used elsewhere? For instance, somewhere else by Oscar? Is it something that we added?